### ISSUE TITLE
How to replace the model in TensorFlow Android Demo

### ISSUE THREAD

**Issue by pd1316** [tensorflow/tensorflow#20492]
I would like to replace the model of the TensorFlowAndroidDemo. There are two files in the asset folder: one is "model.pb", file and one " label.txt".

Now I already have a pretrained pb model. How can I get the corresponding label. txt file?

Thanks

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
Have I written custom code
OS Platform and Distribution
TensorFlow installed from
TensorFlow version
Bazel version
CUDA/cuDNN version
GPU model and memory
Exact command to reproduce

**Comment by skye** [comment]
This question is better asked on [StackOverflow](http://stackoverflow.com/questions/tagged/tensorflow) since it is not a bug or feature request. There is also a larger community that reads questions there.

If you think we've misinterpreted a bug, please comment again with a clear explanation, as well as all of the information requested in the [issue template](https://github.com/tensorflow/tensorflow/issues/new). Thanks!
